packages/functional-tests/src/tests/root-motion-test.ts

```diff
@@ -1,190 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import * as MRESDK from '@microsoft/mixed-reality-extension-sdk';
-import { Actor, AnimationWrapMode } from '@microsoft/mixed-reality-extension-sdk';
-import { promises } from 'fs';
```

Comment:
Is this really deleted?

Reply:
Yes (as we discussed offline, subsumed by `animateTo`)